snaps/how-to/request-permissions.md

```diff
@@ -12,6 +12,10 @@ to communicate with Snaps.
 
 Snaps and dapps follow the [EIP-2255 wallet permissions specification](https://eips.ethereum.org/EIPS/eip-2255).
 
+  :::info note
+  The [JSON-RPC](/wallet/reference/json-rpc-api) methods [`wallet_snap`](/wallet/reference/wallet_snap) and [`wallet_invokeSnap`](/wallet/reference/wallet_invokesnap) are restricted. To use these restricted methods, you must first obtain permission through [`wallet_requestSnaps`](/wallet/reference/wallet_requestSnaps).
+  :::
+
```

Comment:
This is not needed in this location. There is a "Request permissions from a dapp" section at the bottom of this page that mentions `wallet_requestSnaps`. You can add mention of `wallet_snap` and `wallet_invokeSnap` in that section.